lib/iaas/BackupStore.js

```diff
@@ -319,6 +322,15 @@ class Filename {
     );
   }
 
+  /**
+   * 
+   * Checks if the input timestamp is in the following format : yyyy-mm-ddTHH-mm-ssZ
+   **/
+  isValidTimeStamp(timestamp) {
+    var timeStampRegex = new RegExp(`^[0-9]{4}-(0[1-9]|1[012])-(0[1-9]|[12][0-9]|3[01])T(0[0-9]|1[0-9]|2[0-4])-([0-5][0-9])-([0-5][0-9])Z$`);
```

Comment:
use const not 'var'. rather you could also use it like `/^[0-9]{4}-(0[1-9]|1[012])-(0[1-9]|[12][0-9]|3[01])T(0[0-9]|1[0-9]|2[0-4])-([0-5][0-9])-([0-5][0-9])Z$/.test(timestamp)`

Reply:
Done .